apps/app-article/src/components/article-content.js

```diff
@@ -70,14 +82,14 @@ class Content extends Component {
         );
     }
 
-    renderFactBox(block, key) {
+    renderGenericBox(block, key) {
         return (
-            <div className={"factBox"} key={key} id={"factBox-" + key}>
-                <div className={"fakta"}>FAKTA</div>
-                <h4>{block.factBox.title}</h4>
+            <div className={`genericBox genericBox-border-${getBrandValueParam()}`} key={key} id={"genericBox-" + key}>
+                <div className={"genericBox-headline"}>{block.box && block.box.type === "fact" ? 'FAKTA': ''}</div>
```

Comment:
So the logic should be
```
if headline
then headline
else if type == fact
     then "FAKTA"
     else ""
```

Reply:
@vapaj if the current last commit does not fix it, should smth like this be ok:

```
 {
     block.box && block.box.headline &&  
           <div className={"genericBox-headline"}>{block.box.headline}</div>
 }
 {
     block.box && block.box.type &&  
           <div className={"genericBox-headline"}>{block.box.type === "fact" ? 'FAKTA': ''}</div>
 }
```